migration-scripts/contentful-migration.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import requests
+import json
+import time
+
+def make_ids_empty(data):
+    if isinstance(data, dict):
+        if 'sys' in data and 'id' in data['sys']:
+            data['sys']['id'] = ''
+        for key in data:
+            make_ids_empty(data[key])
+    elif isinstance(data, list):
+        for item in data:
+            make_ids_empty(item)
+
+def main(args):
+    headers = {
+        "X-Contentful-Topic": "ContentManagement.Entry.publish"
```

Comment:
Will this mean that everything that is migrated over will all be published by default? This may not be wanted, as things in Contentful that are currently set to archived, deleted etc would end up being visible on the service when we come to interpreting these flags in the application.